test/collection/test_config.py

```diff
@@ -397,6 +397,20 @@ def test_basic_config():
             }
         },
     ),
+    (
+        Configure.Vectorizer.text2vec_weaviate(
+            vectorize_collection_name=False,
+            model="Snowflake/snowflake-arctic-embed-s",
+            base_url="https://inference.weaviate.cloud",
+        ),
+        {
+            "text2vec-weaviate": {
+                "vectorizeClassName": False,
+                "model": "Snowflake/snowflake-arctic-embed-s",
```

Comment:
First supported model is `Snowflake/snowflake-arctic-embed-m-v1.5` so maybe we could change the test to reflect that ?

Reply:
these are only unit tests, so it doesn't matter but I still changed it :)